material/overrides/main.html

```diff
@@ -14,7 +14,7 @@
     <strong>Fosstodon</strong>
   </a>
   and
-  <a href="https://twitter.com/squidfunk">
+  <a href="https://x.com/squidfunk">
     <span class="twemoji twitter">
       {% include ".icons/fontawesome/brands/twitter.svg" %}
```

Comment:
Shouldn't the icon also be changed?
Same in `src/overrides/main.html`

Reply:
Oh, I was thinking of making multiple PRs for changing icons and links and text separately. Will it be OK if this is a massive PR as it will change almost every file TBH

Reply:
I'm not linked with `mkdocs-material`, I was just asking a question 🙂
@squidfunk will know better

Reply:
OK, I will do whichever way he is comfortable with

Reply:
I guess the Twemoji project is now dead, or something last commit was a year ago and their current URL doesn't work, so I will mark this conversation as resolved as nothing can be done about it now. :)

Reply:
There is `x-twitter.svg`, but honestly, it doesn't matter as long as FontAwesome has `twitter.svg`.